src/truncate.jl

```diff
@@ -1,7 +1,41 @@
+"""
+    truncated(d, l, u):
+
+Truncate a distribution between `l` and `u`.
+Builds the most appropriate distribution for the type of `d`,
+the fallback is constructing a `Truncated` distribution.
```

Comment:
How do we call things like `Symmetric`? Wrapper?

Reply:
yes

Reply:
I added the wrapper term